.github/workflows/zips.yml

```diff
@@ -62,6 +62,9 @@ concurrency:
   group: zips-${{ github.ref }}
   cancel-in-progress: true
```

Comment:
Another thing I noticed is that this job ran twice for the same PR (essentially for `pull_request`, and `push`) when we'd only want it to run once in this case.

<img width="394" alt="image" src="https://user-images.githubusercontent.com/1621608/232856552-8c1db4e7-dcae-4f34-afb2-505d24269cb0.png">

This happened because your PR here has the `feature/` prefix which is more of a common convention on other projects, is somewhat of a special case for us and something we only use for long-running feature branches which we'd be targeting PRs _against_. So it's probably not worth going too out of our way to avoid this case as it won't come up often I think, but I would expect the concurrency here to cancel one of them. I'm guessing the `ref` is different depending on a PR or push.